site2/website/scripts/replace.js

```diff
@@ -49,7 +49,7 @@ const versions = getVersions();
 const latestVersion = versions[0];
 
 const from = [
-  /pulsar:version/g, 
+  /{{pulsar:version}}/g, 
```

Comment:
I'm curious as to why the {{}} were added? Should we be consistent with the other variables (pulsar:variable_name)?

Reply:
the main reason I changed to `{{}}` is because of this line: https://github.com/apache/incubator-pulsar/pull/2237/files#diff-2e5bd7bb5ee452df3f4d1c90b6cfff0bL20

it is a bit confusing with `pulsar:pulsar:version`. I thought it is clear to have `{{}}`. I changed `pulsar:version` in this PR for now. if we agreed that is the direction we should head down, I can change the others to make things consistent and clearer.

Reply:
{{}} are commonly used in templating engines so I was trying to avoid using those for search and replace variables. I do see how it's confusing above. Maybe we can do $pulsar:version or <pulsar:version> i'm not sure sure if either of those are are any better, thoughts?

Reply:
$pulsar.version sounds good to me.

Reply:
thinking about it again '$' has meaning in a regex expression. Let's keep the {{}} for now.